client/components/TestReport.jsx

```diff
@@ -59,8 +59,8 @@ const styles = {
 }
 
 // Chart.js setup
-Chart.defaults.global.defaultFontColor = white;
-Chart.defaults.global.defaultFontFamily = 'Roboto';
+// Chart.defaults.global.defaultFontColor = 'white';
+// Chart.defaults.global.defaultFontFamily = 'Roboto';
```

Comment:
Why was this removed?

Reply:
I had an issue with fonts in the chrome dev tools console, UI was broken for me, once I commented out those lines - I was able to see pipelines, not sure if it doesn't work only for me or for others as well.